### PR TITLE
Update examples for calculated fields expressions

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do? What is the motivation?

- Update examples for calculated fields expressions

### Merge instructions

**Do not merge** Pending PM review.

### CODE REVIEW

**Review by brett0000FF — COMMENTED** [review]
Two comments!
* Looks like "Saved Items" in main.en.yaml needs an updated weight.
* Looks like maybe left-nav.html changes were accidentally pulled in from a merge?

File: config/_default/menus/main.en.yaml [hunk 1]
```diff
@@ -5264,41 +5264,46 @@ menu:
...
       parent: log_explorer
-      weight: 512
+      weight: 513
     - name: Watchdog Insights for Logs
       identifier: log_explorer_watchdog_insights
       url: logs/explorer/watchdog_insights/
       parent: log_explorer
-      weight: 513
+      weight: 514
     - name: Saved Views
```
[the remark this answers is not in the document]
Comment (estherk15) [reply config/_default/menus/main.en.yaml]: Thanks for catching this, I nested it as a 4th level, but forgot to change the weights back 🫠

**Comment by estherk15** [comment]
> Two comments!
> 
> * Looks like "Saved Items" in main.en.yaml needs an updated weight.
> * Looks like maybe left-nav.html changes were accidentally pulled in from a merge?

Thanks for the review @brett0000FF, I messed with my VSCode settings and realized it was changing files on save 😬

**Review by brett0000FF — APPROVED** [review]
🚀

**Review by aaxzheng — APPROVED** [review]

**Pull request merged**